tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -1131,6 +1164,16 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
           ->set_output_to_operand_aliasing({{{}, {2, {}}}});
     }
     TF_RETURN_IF_ERROR(SetName(instr->GetModule(), fused_op.get()));
+    if (slice != nullptr) {
+      int slice_op_dim = slice->operand(0)->shape().rank();
+      if (slice->slice_starts() != std::vector<int64_t>(slice_op_dim, 0)) {
+        return OkStatus();
+      }
```

Comment:
Check that the strides are 1. Also better to move this check to the start of the function, so you don't unnecessarily create extra ops. Also add a comment stating "Only slices which chop off the ends of dimensions are supported".

Reply:
Resolved.